### PR TITLE
Chore-creating-redflag-routes

### PR DESCRIPTION
**Pull request by NicholusMuwonge**
What the PR is about?

Added redflag records routes to my existing intervention routes.

 Description of the tasks to be completed:

- [x] refactoring tests

- [x] being able to get redflags , edit their status by administrator, and deleting a redflag.

- [x] changing the tests database

How should this be manually tested:

- Take a look at my updated pull request.

Please, kindly review:
@habibsentongo @rhytah @mastula77 @richiem @wasswa-derick

### CODE REVIEW

**Comment by HabibSentongo** [comment]
great, this is good work and keep the pace.

File: tests/test.py [hunk 1]
```diff
@@ -184,14 +208,97 @@ def test_post_user_empty_fields(self):
...
+    def test_invalid_password(self):
+        """
+        Test for password less than 5 characters
+        :return:
+        """
+        register = self.client.post('/api/v2/auth/signup/',\
+            data = json.dumps(dict(user_name = 'milbeat',user_password = 'str',email = 'milly@gamil.com')),\
+            content_type = ('application/json'))
+        response_data = json.loads(register.data.decode())
+        print (response_data)
```
Comment (wasswa-derick) [inline tests/test.py:293]: @NicholusMuwonge Does this print statement add value to the test.
If not, kindly consider removing it.

File: tests/test.py [hunk 2]
```diff
@@ -184,14 +208,97 @@ def test_post_user_empty_fields(self):
...
+        self.assertTrue(response_data['status'], 'fail')
+        self.assertTrue(response_data['error_message'],
+                        'Password is wrong. It should be \
+            at-least 5 characters long, and alphanumeric.It shouldnt also be \
+            longer than 13 characters')
+        # self.assertFalse(response_data['data'])
```
Comment (wasswa-derick) [inline tests/test.py:299]: @NicholusMuwonge Kindly consider removing all the commented code. Commenting it out simply indicates that it is not needed.

File: tests/test.py [hunk 3]
```diff
@@ -152,30 +174,32 @@ def test_post_user_empty_fields(self):
...
+        request_data = self.client.post('/api/v2/auth/login/',\
+        data = json.dumps({"user_name":"nichol","user_password":''}),\
+        content_type = ('application/json'))
+        response_data = json.loads(request_data.data.decode())
+        print (response_data)
+        self.assertEqual(request_data.status_code,400)
+        self.assertTrue(request_data.content_type,'application/json')
+        self.assertTrue(response_data['error_message'],'wrong password')
 
     # def test_get_all_users_in_empty_list(self):
```
Comment (wasswa-derick) [inline tests/test.py:204]: @NicholusMuwonge Kindly consider removing all the commented code from your pull request.

File: tests/test.py (hunk 1 above)
Comment (NicholusMuwonge) [reply tests/test.py:293]: okay let me update that, please

File: tests/test.py (hunk 3 above)
Comment (NicholusMuwonge) [reply tests/test.py:204]: yes please it will be out in the next commit

**Pull request merged**